src/main/kotlin/solve/importer/view/ProjectTreeView.kt

```diff
@@ -50,7 +57,19 @@ open class ProjectTreeView : View() {
                 private val errorFolderIcon = loadImage("icons/importer/error_folder.png")
                 private val errorFileIcon = loadImage("icons/importer/error_file.png")
 
+                private val imageIconDark = loadImage("icons/importer/photo_dark_theme.png")
```

Comment:
Rewrite image loading the same way as it completed in [PR](https://github.com/prime-slam/SOLVE/pull/111)